src/VideoTransportSlider/VideoTransportSlider.js

```diff
@@ -454,6 +456,9 @@ module.exports = kind(
 		this.selected = false;
 		this.removeClass('visible');
 		this.endPreview();
+		if (gesture.downEvent) {
+			gesture.up(util.clone(gesture.downEvent));
```

Comment:
I'm not sure we need to clone the `downEvent` here, as we are effectively doing this in `up` already: https://github.com/enyojs/enyo/blob/2.6.0-dev/src/gesture/gesture.js#L82. Semantically, it seemed slightly weird to send the `downEvent` to `up`, but not sure of an alternative, as it seems to be the only property that contains the relevant properties specific to the event originator.

Also, along the lines of @Djspaceg's feedback, I think we could make this more general and move this to Spotlight's `onEvent` handler, similar to how we are calling `gesture.drag.endHold()` for the `blur` event.